grid/chat/channels/maps/slack.test.ts

```diff
@@ -1,3 +1,9 @@
 import { getChannelsTest } from './channels';
 
-getChannelsTest('slack');
+/**
+ * Live tests
+ *
+ * @group live/safe
+ */
+
+getChannelsTest('slack', []);
```

Comment:
question: why there are empty destinations?

Reply:
Ok, i see it now. `channels` test helper now feels little bit forced - discord and slack are not so similar :/

Reply:
yeah, I can change it to optional parameter